viniciuslima-99/introduction-to-web-development
# Feature/server-base

### What I did

- Changed the README.md file
- Feature(server-base) : Added the .gitignore file
- Started the project node, add some dependencies and changed the start on scripts
- Added some dependencies on project, the express and nodemon
- Added the server.js file and configured the express, a route (/) and enabled a port for the server.

### How to test

1. Go to the server folder on the terminal execute the npm init command
2. Go to localhost: 3000 and see if it returns the message " Respondendo a pagina web na rota / "
3. Go to the terminal where the project is being executed and see if it returns the message "Respondendo o terminal na rota / "
